src/Sylius/Component/Core/Dashboard/SalesDataArrayNormalizerInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Dashboard;
+
+interface SalesDataArrayNormalizerInterface
+{
+    public function completeNoSalesMonthData(
+        \DateTimeInterface $startDate,
+        \DateTimeInterface $endDate,
+        array $salesData
+    ): array;
```

Comment:
I'm not feeling comfortable with adding this interface to the core - it takes an array and returns an array, looks like it could only be used by `SalesDataProvider`. I'd include its implementation in a private method there and not create any more abstractions at this time, let's see how it's used first.

Reply:
Adding this interface later is fine, removing it is a BC break :)

Reply:
Hmm, what if this responsibility would be moved to `SalesSummary` while creating it? `SalesSummary::createForPeriod($startDate, $endDate)->add($year, $month, $total)`?